### PR TITLE
Chore/1.3.0

### PR DESCRIPTION
**Pull request by Baroshem**
## Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes (if not applicable, please state why)

### CODE REVIEW

File: docs/content/1.documentation/5.advanced/2.faq.md [hunk 1]
```diff
@@ -376,3 +376,26 @@ Alternatively, you can use the `external` attribute on `NuxtLink` to set the nav
...
+
+### Running app with `--host` flag
+
+If you want to expose your app in local network to test it by using other devices you can do so by adding the following configuration:
+
+```ts
+security: {
+    headers: {
+      crossOriginEmbedderPolicy: process.env.NODE_ENV === 'development' ? 'unsafe-none' : 'require-corp', //https://github.com/Baroshem/nuxt-security/issues/101
+      strictTransportSecurity: false,
```
Comment (vejja) [inline docs/content/1.documentation/5.advanced/2.faq.md:388]: strictTransportSecurity: true is better and will work

**Review by vejja — CHANGES_REQUESTED** [review]

File: docs/content/1.documentation/5.advanced/2.faq.md (hunk 1 above)
Comment (Baroshem) [inline docs/content/1.documentation/5.advanced/2.faq.md:388]:
```suggestion
      strictTransportSecurity: true,
```

Comment (Baroshem) [reply docs/content/1.documentation/5.advanced/2.faq.md]: Added

**Review by vejja — APPROVED** [review]

**Comment by Baroshem** [comment]
Will add few more features progably before releasing.

**Review by dammy001 — APPROVED** [review]

**Pull request merged**